### PR TITLE
Feature/post service

### PR DESCRIPTION
**Pull request by FoeverNa**
please

### CODE REVIEW

File: src/test/java/com/fastcampus/blog/service/PostServiceTest.java [hunk 1]
```diff
@@ -16,6 +22,28 @@
     @Mock
     private PostRepository postRepository;
 
+    @Test
+    void getPostsTest(){
+        when(postRepository.findAll()).thenReturn(mockPosts());
+
+        List<Post> result = postService.getPosts();
+
+        System.out.println(">>>" + result);
```
Comment (limjoonchul) [inline src/test/java/com/fastcampus/blog/service/PostServiceTest.java]: remove print

File: src/test/java/com/fastcampus/blog/service/PostServiceTest.java [hunk 2]
```diff
@@ -66,6 +66,16 @@ void updatePostTest() {
...
+
+    }
+
+    @Test
+    void deletePostTest() {
+        Post result = postService.deletePost(1L);
+
+        verify(postRepository, times(1)).findPostById(1L);
+        verify(postRepository, times(1)).save(any(Post.class));
+//        assertThat(result.isDeleted()).isEqualTo(true);
```
Comment (limjoonchul) [inline src/test/java/com/fastcampus/blog/service/PostServiceTest.java]: remove annotation

**Review by limjoonchul — CHANGES_REQUESTED** [review]

File: src/test/java/com/fastcampus/blog/service/PostServiceTest.java (hunk 1 above)
Comment (FoeverNa) [reply src/test/java/com/fastcampus/blog/service/PostServiceTest.java]: deleted

File: src/test/java/com/fastcampus/blog/service/PostServiceTest.java (hunk 2 above)
Comment (FoeverNa) [reply src/test/java/com/fastcampus/blog/service/PostServiceTest.java]: already deleted

**Pull request merged**